### PR TITLE
Extract different views

### PR DESCRIPTION
**Pull request by peculiarity**

### CODE REVIEW

**Comment by peculiarity** [comment]
@maciejka @krzkaczor

File: cypress/pages/Finalization.ts [hunk 1]
```diff
@@ -4,6 +4,21 @@ import { Summary } from './Summary';
...
   public currentTx: string = '';
 
+  public shoulHavePriceImpactWarning = () => {
+    cy.get(tid('price-impact-text'))
+      .contains(/Order has a significant .../);
+
+    return this;
+  }
+
+  public acceptPriceImpact = () => {
```
Comment (krzkaczor) [inline cypress/pages/Finalization.ts]: return this? to make it consistent?

File: cypress/pages/Finalization.ts [hunk 2]
```diff
@@ -4,6 +4,21 @@ import { Summary } from './Summary';
...
+    cy.get(tid('proceed-with-order')).click();
+  }
+
+  public dismissPriceImpact = () => {
+    cy.get(tid('dismiss-warning')).click();
```
Comment (krzkaczor) [inline cypress/pages/Finalization.ts]: same

File: src/blockchain/config.ts [hunk 3]
```diff
@@ -44,8 +44,8 @@ export const tokens = asMap('symbol', [
     maxSell: '10000000',
     name: 'Wrapped Ether',
     icon: SvgImageSimple(ethSvg),
-    iconInverse: SvgImageSimple(ethInverseSvg),
-    iconCircle: SvgImageSimple(ethCircleSvg),
+    iconInverse: SvgImageSimple(ethCircleSvg),
```
Comment (krzkaczor) [inline src/blockchain/config.ts]: I think this change impacts icons on notifications (see vis reg for storybook)

File: src/instant/CurrentPrice.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  quotation?: string
+}> {
+  public render() {
+    const { price, quotation } = this.props;
+    return (
+      <div className={classnames(styles.details, styles.finalization)}>
+        <span>Current Estimated Price</span>
+        <span style={{ marginLeft: '12px', color: '#828287' }}>
+          <Approximate>
+            {price ? formatAmount(price, 'USD') : ''}
```
Comment (krzkaczor) [inline src/instant/CurrentPrice.tsx]: you could use `price && formatAmount(price, 'USD')`

File: src/blockchain/config.ts (hunk 3 above)
Comment (peculiarity) [reply src/blockchain/config.ts]: Actually it's not broken. WETH should have this icon. It's the inversed icon of ETH. It's all good.

image

image

Comment (krzkaczor) [reply src/blockchain/config.ts]: I was refering to this: https://artifacts.codechecks.io/166230898/f42cfb87b3de11d853d299b0c926404a4f958d9a/storybook-vis-reg-report/index.html

**Pull request merged**